administration/src/bp-modules/cards/AddCardsForm.tsx

```diff
@@ -77,29 +54,43 @@ const AddCardsForm = ({
   }
 
   return (
-    <Scrollable>
-      <FormsWrapper>
+    <Stack sx={{ flexGrow: 1, overflow: 'auto', justifyContent: 'safe center', alignItems: 'center' }}>
+      <Stack
+        sx={{
+          flexDirection: 'row',
+          flexWrap: 'wrap',
+          justifyContent: 'center',
+          alignItems: 'flex-start',
```

Comment:
:thinking: I think it looks better when it's centerd.
:question:  And why do you prefer inline styling?
```suggestion
          alignItems: 'center',
```

Reply:
> 🤔 I think it looks better when it's centerd.

I'll change it, I have no strong preference on this one

> ❓ And why do you prefer inline styling?

Several reasons:
- When using styled components or similar, you always end up with stupid names like `StyledFoo` or `CustomBar` that carry no meaning, but you need to have a one, otherwise it does not work.
- This form is actually shorter for many occasions.
- No jumping around when editing a layout.
- This is one of the preferred styling methods for one-off MUI components.
- The styles get compiled into a css class module, so it's not actually "inline" styling in the html/css sense.